### PR TITLE
add xz / lzma library to docker files

### PR DESCRIPTION
**Pull request by jschaul**
in preparation for https://github.com/wireapp/wire-server/pull/1270

### CODE REVIEW

File: build/alpine/Dockerfile.prebuilder [hunk 1]
```diff
@@ -34,7 +34,7 @@ RUN apk add --no-cache \
         snappy-dev \
         llvm-libunwind-dev \
         bash \
-        xz \
+        xz-dev \
```
Comment (akshaymankar) [inline build/alpine/Dockerfile.prebuilder:37]: I think we need `xz` for `ghcup` to work.

File: build/alpine/Dockerfile.deps [hunk 2]
```diff
@@ -27,4 +27,5 @@ RUN apk add --no-cache \
             llvm-libunwind \
             ca-certificates \
             dumb-init \
+            xz \
```
Comment (akshaymankar) [inline build/alpine/Dockerfile.deps:30]: I looked for liblzma.so.5 and found it in `xz-utils`:
https://pkgs.alpinelinux.org/contents?file=liblzma.so.5&path=&name=&branch=v3.12
TBH, I don't really know the difference between liblzma.so and liblzma.so.5, but my locally compiled binaries depend on `liblzma.so.5`.

**Review by akshaymankar — CHANGES_REQUESTED** [review]

**Comment by akshaymankar** [comment]
I think we don't need this PR because the tests are green for alpine builds for #1270

File: build/alpine/Dockerfile.prebuilder (hunk 1 above)
Comment (jschaul) [reply build/alpine/Dockerfile.prebuilder]: I thought one included the other, apparently not. Thanks for catching this issue, I now include both.

File: build/alpine/Dockerfile.deps (hunk 2 above)
Comment (jschaul) [reply build/alpine/Dockerfile.deps]: `xz-libs` is what you meant to write, yes, I added that (though it seems to already get included by something else and is present already in the alpine-deps image).

Comment (akshaymankar) [reply build/alpine/Dockerfile.deps]: > xz-libs is what you meant to write
yes, not enough coffee!

Did you see my other comment? Maybe we don't need this PR, the pipeline is happy.

Comment (jschaul) [reply build/alpine/Dockerfile.deps]: I thought explicit is better than not explicit, but ok :)

**Pull request closed without merging**